README.md

```diff
@@ -235,20 +186,23 @@ func connect() {
     if err != nil {
         // ... handle error
     }
+    // call cleanup when you're done with the database connection
     defer cleanup()
 
     db, err := sql.Open(
         "cloudsql-mysql",
-        "myuser:mypass@cloudsql-mysql(my-project:us-central1:my-instance)/mydb",
-	)
+        "myuser:mypass@cloudsql-mysql(project:region:instance)/mydb",
+    )
     // ... etc
 }
 ```
 
 ### SQL Server
```

Comment:
I think this should be an h4 to be the same as `MySQL` and `Postgres`?
```suggestion
#### SQL Server
```

Reply:
Yes. Thanks for checking these.